hs_core/templates/pages/my-resources.html

```diff
@@ -34,7 +34,7 @@ <h2 class="page-title">Resources</h2>
                     <th>Title</th>
                     {# <th><i class="text-warning fa fa-star"></i></th> #}
                     <th>Type</th>
-                    <th>Owner</th>
+                    <th>First author</th>
```

Comment:
so we're still calling these Owners in the template above, but "First author"/"author" here?